src/extension.ts

```diff
@@ -57,12 +57,13 @@ async function openWindowsTerminal(profile: IWTProfile, uri?: vscode.Uri) {
     if (uri.authority.startsWith('ssh-remote+')) {
       // Handle SSH remotes first
       const host = uri.authority.split('+')[1];
-      const isWindows = !!uri.path.match(/^\/[a-z]:\//);
+      const isWindows = !!uri.path.match(/^\/[a-zA-Z]:\/.*$/);
+      const remoteMachine = await resolveSSHHostName(host);
       if (isWindows) {
-        // Changing paths on Windows seems tricky
-        args.push('ssh', host);
+        // remove first /
+        const uri_windows = uri.path.substring(1, uri.path.length);
+        args.push('ssh', remoteMachine, `cd ${uri_windows} && powershell.exe`);
```

Comment:
I don't think this is reliable enough unfortunately as `&&` may not work in the shell. sshing in will either run in cmd or powershell and from my experiments I couldn't figure out how to link 2 commands together in cmd with ssh liking it. Something I have yet to try is wrapping it in a `powershell -Command` to re-launch the shell which might work.

Reply:
i think we should probably use cmd.exe /c "cd ... && shell" wt.exe creates multiples panels when sees a ; even inside a string. i will investigate this.